### PR TITLE
TOOLS-162 add build docker image for a service

### PR DESCRIPTION
**Pull request by psihachina**
## Changelog: 
- add build docker image for a service 

## Test: 
[asciicast](https://asciinema.org/a/HDBSEwpADLQypeplYdig3tN8z)

### CODE REVIEW

File: internal/commands/deploy.go [hunk 1]
```diff
@@ -25,7 +31,88 @@ func (b *commandsBuilder) newDeployCmd() *deployCmd {
...
+			if err != nil {
+				return err
+			}
+			cc.log.Infof("%s config: %s", serviceName, sConfig)
+
+			if sConfig.Path == "" {
+				return fmt.Errorf("project path not set")
+			}
+
+			dockerRegistry := fmt.Sprintf("%v.dkr.ecr.%v.amazonaws.com", *resp.Account, cc.config.AwsRegion)
```
Comment (igorkotof) [inline internal/commands/deploy.go:70]: can we add a way to override it when needed b/c AWS has public docker repo as well like `public.ecr.aws/r6g9m2o3/news-blog:latest`

File: internal/commands/deploy.go [hunk 2]
```diff
@@ -25,7 +31,88 @@ func (b *commandsBuilder) newDeployCmd() *deployCmd {
...
+			err = ecsdeploy.Build(cc.log, ecsdeploy.Option{
+				Tags: []string{
+					dockerImageName,
+					fmt.Sprintf("%s/%s:%s", dockerRegistry, dockerImageName, strings.Trim(tag, "\n")),
+					fmt.Sprintf("%s/%s:%s", dockerRegistry, dockerImageName, tagLatest),
+				},
+				Dockerfile: dockerfile,
+				BuildArgs: map[string]*string{
+					"DOCKER_REGISTRY":   &dockerRegistry,
+					"DOCKER_IMAGE_NAME": &dockerImageName,
```
Comment (igorkotof) [inline internal/commands/deploy.go:104]: What about an additional arg: BUILDKIT_INLINE_CACHE ?

File: internal/commands/deploy.go (hunk 1 above)
Comment (psihachina) [reply internal/commands/deploy.go:70]: of course, the current is not in this PR

File: internal/commands/deploy.go (hunk 2 above)
Comment (psihachina) [reply internal/commands/deploy.go:104]: It seems like he is not needed now

**Review by AutomationD — APPROVED** [review]

**Pull request merged**